hack/verify-godeps.sh

```diff
@@ -128,7 +128,12 @@ pushd "${KUBE_ROOT}" 2>&1 > /dev/null
     echo "If you're seeing this locally, run the below command to fix your Godeps.json:"
     echo "patch -p0 < godepdiff.patch"
     echo "(The above output can be saved as godepdiff.patch if you're not running this locally)"
+    echo "(The patch file should also be exported as a build artifact if run through CI)"
     KEEP_TMP=true
+    if [[ -x godepdiff.patch && -d "${WORKSPACE}/_artifacts" ]]; then
```

Comment:
I don't think that is set in https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/verify-dockerized.sh?